spec/account/account-service.spec.js

```diff
@@ -47,6 +47,7 @@ describe("AccountService", () => {
                 expect(result).toEqual({
                     accountId: "unhashedAccountId"
                 });
+                expect(storageMock.delAsync).toHaveBeenCalledWith("account/hashedContent");
```

Comment:
This test should be changed to reflect that you're dealing with deleting the registration information, not the account.